### PR TITLE
New Load Balancer Profile for the Network Activity classes

### PR DESCRIPTION
**Pull request by nathanbvail**
#### Description of changes: 
Changes are a result of the conversations in previous PR [885](https://github.com/ocsf/ocsf-schema/pull/885). 

Changes are: 
- New `Load Balancer` profile that can be applied across the `Network Activity` classes. The profile contains a singular `load_balancer` attribute. 
- New `load_balancer` object.
- New `endpoint_connection` object.
- Updates to the dictionary to include `load_balancer`, `endpoint_connection`, and creation of `network_endpoint` 

Attached screenshots: 
image
image

### CODE REVIEW

File: dictionary.json [hunk 1]
```diff
@@ -1343,6 +1343,12 @@
       "description": "The end time of a time period. See specific usage.",
       "type": "timestamp_t"
     },
+    "endpoint_connection": {
+      "caption": "Endpoint Connection",
+      "description": "Contains information about a network connection attempt. See specific usage.",
```
Comment (Aniak5) [inline dictionary.json:1348]: These should be plural `endpoint_connections` for the attribute name and `Endpoint Connections` for the `caption`.

Comment (nathanbvail) [reply dictionary.json:1348]: This dang definition keeps tricking me up!

File: objects/load_balancer.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        "requirement": "recommended"
+        },
+    "code": {
+        "caption": "Response Code",
+        "description": "The numeric response status code detailing the connection from the load balancer to the destination target.",
+        "requirement": "recommended"
+        },
+    "endpoint_connection":{
+        "caption": "Endpoint Connection",
+        "description": "An object detailing the load balancer connection attempts and responses.",
```
Comment (Aniak5) [inline objects/load_balancer.json:26]: These also need to be plural.

**Review by mikeradka — APPROVED** [review]

**Comment by pagbabian-splunk** [comment]
I don't see the plural of `endpoint_connections` called out by Ania.

File: objects/load_balancer.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        "caption": "Destination Endpoint",
+        "description": "The destination to which the load balancer is distributing traffic.",
+        "requirement": "recommended"
+        },
+    "code": {
+        "caption": "Response Code",
+        "description": "The numeric response status code detailing the connection from the load balancer to the destination target.",
+        "requirement": "recommended"
+        },
+    "endpoint_connections":{
```
Comment (nathanbvail) [inline objects/load_balancer.json:24]: @pagbabian-splunk Here is one plural correction

File: dictionary.json [hunk 4]
```diff
@@ -1343,6 +1343,12 @@
       "description": "The end time of a time period. See specific usage.",
       "type": "timestamp_t"
     },
+    "endpoint_connections": {
```
Comment (nathanbvail) [inline dictionary.json:1346]: @pagbabian-splunk  Second plural correction

**Review by Aniak5 — APPROVED** [review]
Looks great @nathanbvail, nice work!

**Review by pagbabian-splunk — APPROVED** [review]

**Pull request merged**